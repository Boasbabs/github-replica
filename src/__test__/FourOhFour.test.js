/**
 * TODO: 404 Snapshot test
 */
import React from "react";
import Enzyme, { render, shallow } from "enzyme";
import enzymeAdapterReact16 from "enzyme-adapter-react-16";
import FourOhFour from "../components/fourOhfour";
import { BrowserRouter as Router } from "react-router-dom";

Enzyme.configure({ adapter: new enzymeAdapterReact16() });

const getDefaultProps = () => ({});

describe("FourOhFour component", () => {
  it("it matches snapshot", () => {
    const {} = getDefaultProps();
    const wrapper = shallow(
      <Router>
        <FourOhFour />
      </Router>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
