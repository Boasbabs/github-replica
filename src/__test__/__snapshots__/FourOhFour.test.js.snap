// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour component it matches snapshot 1`] = `ShallowWrapper {}`;
